pkg/grid/src/state/kiln.ts

```diff
@@ -33,6 +43,13 @@ const useKilnState = create<KilnState>((set, get) => ({
     const lag = await api.scry<boolean>(scryLag);
     set({ lag });
   },
+  toggleInstall: async (desk: string, ship: string) => {
+    const synced = !!get().pikes[desk].sync;
+    await (useMockData
+      ? fakeRequest('')
+      : api.poke(synced ? kilnUninstall(desk) : kilnInstall(ship, desk)));
```

Comment:
(Whoops, comment didn't save)

For the System Updates case, this needs to install the foreign %kids desk to the local %base desk.  Looks like `kilnInstall` supports this with the optional `local?` argument.

https://github.com/urbit/urbit/blob/1bd0f4be77fcdd6c69799ac78429b08e3552b9d0/pkg/npm/api/hood/lib.ts#L12-L16

Reply:
Good catch, updated the `kilnInstall` usage to match the previous implementation:

https://github.com/urbit/urbit/blob/4834ec59a2a7d73969aa831a92d2c8ad232bd5e4/pkg/grid/src/state/kiln.ts#L59